software/py/vanilla_parser/stats_parser.py

```diff
@@ -215,10 +215,10 @@ def __init__(self, per_tile_stat, per_tile_group_stat, per_vcache_stat, vanilla_
         self.stats, self.instrs, self.misses, self.stalls, self.bubbles = self.parse_header(vanilla_input_file)
 
 
-        # bubble_fp_op is a bubble in the Integer pipeline "caused" by
+        # bubbles that are  a bubble in the Integer pipeline "caused" by
         # an FP instruction executing. Don't count it in the bubbles
         # because the procesor is still doing "useful work". 
-        self.notbubbles = ['bubble_fp_op'] 
+        self.notbubbles = [] 
```

Comment:
Just curious, is this changed fp ops won't create bubbles in integer pipeline anymore?

Reply:
No, originally bubble_fp_op was just a bubble in the Integer pipeline but the FP pipeline was busy, so we didn't use to count it as bubbles. But in the new version bubble_fp_op is eliminated all together, so no need to have it in the parser anymore.